modules/osd-aws-privatelink-firewall-prerequisites.adoc

```diff
@@ -3,27 +3,40 @@
 // * osd_planning/aws-ccs.adoc
 // * rosa_install_access_delete_clusters/rosa_getting_started_iam/rosa-aws-prereqs.adoc
 // * rosa_planning/rosa-sts-aws-prereqs.adoc
-// * rosa_planning/rosa-hcp-prereqs.adoc
 
 ifeval::["{context}" == "rosa-sts-aws-prereqs"]
 :fedramp:
+:rosa-classic-sts:
 endif::[]
-ifeval::["{context}" == "rosa-hcp-aws-prereqs"]
-:fedramp:
+ifeval::["{context}" == "aws-ccs"]
+:osd:
+endif::[]
+ifeval::["{context}" == "prerequisites"]
+:rosa-classic:
 endif::[]
 
 :_mod-docs-content-type: PROCEDURE
+ifdef::rosa-classic-sts[]
+[id="rosa-classic-firewall-prerequisites_{context}"]
+= ROSA Classic
+endif::rosa-classic-sts[]
+ifndef::rosa-classic-sts[]
 [id="osd-aws-privatelink-firewall-prerequisites_{context}"]
 = AWS firewall prerequisites
 
+This section provides the necessary details that enable you to control egress traffic from your {product-title} cluster. If you are using a firewall to control egress traffic, you must configure your firewall to grant access to the domain and port combinations below. {product-title} requires this access to provide a fully managed OpenShift service.
```

Comment:
I think we are supposed to avoid self-referential writing (i.e. "This document..." or "This tutorial..."). Would something along the following lines work?
```suggestion
If you are using a firewall to control egress traffic from your {product-title}, you must configure your firewall to grant access to certain domain and port combinations. {product-title} requires this access to provide a fully managed OpenShift service.
```